code/modules/ruins/objects_and_mobs/ash_walker_den.dm

```diff
@@ -31,6 +31,7 @@
 			for(var/obj/item/W in H)
 				H.unEquip(W)
 			H.gib()
+			health = health + 10//restores 5% hp of tendril
```

Comment:
adjustHealth(-10)
or if you want to get fancy, adjustHealth(-maxHealth * 0.05), which is always 5% regardless of maxhealth